src/pages/assets.tsx

```diff
@@ -117,7 +118,7 @@ const AssetsPage = ({ data }) => {
           <AssetDownload
             title={translateMessageId("page-assets-hero", intl)}
             alt={translateMessageId("page-assets-hero", intl)}
-            image={data.hero}
+            image={data.hero as Record<string, unknown>}
```

Comment:
@pettinarip Type assetion here is for the AssetDownload component prop to don't cry. It actually complains a lot because the props of the component expect a non-nullable object, even tho in the component this specific prop value is used for a gatsby call is using getSrc, a null-safe method ([docs here](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#getimage))

Reply:
This is fine but there are some issues with the `getImage` types. In the `overrides.d.ts` we are overriding the types for this function to basically accepts `any` as its argument. So, in order to keep it simple, we can just accept `any` in `AssetDownload` while we wait for a proper fix.